hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -128,6 +129,14 @@ public class DBScanner implements Callable<Void>, SubcommandWithParent {
           "eg.) \"name,acls.type\" for showing name and type under acls.")
   private String fieldsFilter;
 
+  @CommandLine.Option(names = {"--filter"},
+      description = "Comma-separated list of \"<field>:<operator>:<value>\" where " +
+          "<field> is any valid field of the record, " +
+          "<operator> is (EQUALS,MAX or MIN) and " +
+          "<value> is the value of the field. " +
+          "eg.) \"dataSize:equals:1000\" for showing records having the value 1000 for dataSize")
```

Comment:
How does multiple matches work? If we have multiple fields called `size` or `version` or `name`, how will the visual representation tell me which field is being filtered? How does `MIN` and `MAX` work with strings?

Reply:
As a class can't have 2 fields of the same name, we won't encounter this at the same level of the hierarchy. If you mean something like this:
```
{ 
name : <value>,
acls: { type : <value>,
        name : <value>,
      }
}
```
Then we would be passing the option like this for the full name of the field: 
`--filter="name:equals:xyz"` and 
`--filter="acls.name:equals:xyz"`
In the result, the full record will be shown same as how a normal scan command without the filter is seen.